homeassistant/components/vegehub/coordinator.py

```diff
@@ -0,0 +1,24 @@
+"""Coordinator for the Vegetronix VegeHub."""
+
+from __future__ import annotations
+
+import logging
+
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.update_coordinator import DataUpdateCoordinator
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class VegeHubCoordinator(DataUpdateCoordinator):
```

Comment:
```suggestion
class VegeHubCoordinator(DataUpdateCoordinator[data type]):
```
this is missing the type for `self.data` here

Reply:
Fixed in 35fa1aa59ef169d3c56b061e7112fd4c77e2a5fa